librz/bin/p/bin_pe.inc

```diff
@@ -717,3 +725,25 @@ static RzList *compute_hashes(RzBinFile *bf) {
 static RzList *strings(RzBinFile *bf) {
 	return rz_bin_file_strings(bf, 4, false);
 }
+
+/**
+ * Currently only used by CIL plugin to resolve `call` and `jmp` targets
+ * \returns offset on success, UT64_MAX on fail
+ */
+static ut64 get_offset(RzBinFile *bf, int type, int idx) {
+	struct PE_(rz_bin_pe_obj_t) * bin;
+	Pe_image_clr *clr;
+
+	if (!bf || !bf->o ||
+		!(bin = bf->o->bin_obj) ||
+		!(clr = bin->clr)) {
+		return UT64_MAX;
+	}
+
+	if (type == 'd') { // MethodDef index
+		Pe_image_metadata_methoddef *methoddef = rz_list_get_n(clr->methoddefs, idx - 1);
```

Comment:
another note: using rz_list in this way is kinda slow, because it will loop till `n` equals `idx - 1`, so this is like a big bottleneck.

Reply:
Makes sense. I'll switch to PVectors